app/buyers/views/buyers.py

```diff
@@ -270,59 +279,229 @@ def view_brief_responses(framework_slug, lot_slug, brief_id):
+        self.content_loader = kwargs.pop('content_loader', content_loader)
+
+        super(View, self).__init__(**kwargs)
+
+    def get_responses(self, brief):
+        return get_sorted_responses_for_brief(brief, self.data_api_client)
+
+    def get_context_data(self, **kwargs):
+        get_framework_and_lot(kwargs['framework_slug'], kwargs['lot_slug'],
+                              self.data_api_client, status='live',
+                              must_allow_brief=True)
+
+        brief = self.data_api_client.get_brief(kwargs['brief_id'])["briefs"]
+
+        if not is_brief_correct(brief, kwargs['framework_slug'],
+                                kwargs['lot_slug'], current_user.id):
+            abort(404)
+
+        if brief['status'] != "closed":
+            abort(404)
+
+        kwargs.update({
+            'brief': brief,
+            'responses': self.get_responses(brief),
+        })
+
+        return kwargs
+
+    def get_questions(self, framework_slug, lot_slug, manifest):
+        section = 'view-response-to-requirements'
+
+        result = self.content_loader.get_manifest(framework_slug, manifest)\
+                                    .filter({'lot': lot_slug})\
+                                    .get_section(section)
+
+        return result.questions if result else []
+
+    def create_csv_response(self, context=None):
+        column_headings = []
+        question_key_sequence = []
+        boolean_list_questions = []
+        csv_rows = []
+        brief = context['brief']
+
+        questions = self.get_questions(context['framework_slug'],
+                                       context['lot_slug'],
+                                       'legacy_output_brief_response')
+
+        # Build header row from manifest and add it to the list of rows
+        for question in questions:
+            question_key_sequence.append(question.id)
+            if question['type'] == 'boolean_list' and brief.get(question.id):
+                column_headings.extend(brief[question.id])
+                boolean_list_questions.append(question.id)
+            else:
+                column_headings.append(question.name)
+        csv_rows.append(column_headings)
+
+        # Add a row for each eligible response received
+        for brief_response in context['responses']:
+            if all_essentials_are_true(brief_response):
+                row = []
+                for key in question_key_sequence:
+                    if key in boolean_list_questions:
+                        row.extend(brief_response.get(key))
+                    else:
+                        row.append(brief_response.get(key))
+                csv_rows.append(row)
+
+        return Response(
+            csv_generator.iter_csv(csv_rows),
+            mimetype='text/csv',
+            headers={
+                "Content-Disposition": (
+                    "attachment;filename=responses-to-requirements-{}.csv"
+                ).format(brief['id']),
```

Comment:
True, consistency would be nice. I'd suggest checking with @tracy-content regarding file names. It looks like nothing has yet been confirmed (or at least written on the pivotal) so we should check if it should remain as the CSV or we would prefer something else: https://www.pivotaltracker.com/story/show/133916871